site/content/docs/0.0/about/license.md

```diff
@@ -35,4 +35,4 @@ OUDS Web is released under the MIT license and is copyright {{< year >}} Orange
 - Include the source of OUDS Web nor Bootstrap itself, or of any modifications you may have made to it, in any redistribution you may assemble that includes it
 - Submit changes that you make to OUDS Web or Bootstrap back to their respective project (though such feedback is encouraged)
 
-For more information, the full OUDS Web license is located [in the project repository]({{< param repo >}}/blob/v{{< param current_version >}}-ouds-web/LICENSE), and Bootstrap license is located [in the project repository](https://github.com/twbs/bootstrap/blob/v{{< param bootstrap_current_version >}}/LICENSE).
+For more information, the [full OUDS Web license is located in the project repository]({{< param repo >}}/blob/v{{< param current_version >}}-ouds-web/LICENSE), and [Bootstrap license is located in the project repository](https://github.com/twbs/bootstrap/blob/v{{< param bootstrap_current_version >}}/LICENSE).
```

Comment:
you mean use "Bootstrap license is located in the project repository" as the link label instead of "in the project repository"? But we have no need to do this in Bootstrap because there is only one link...

Reply:
No I mean here: https://boosted.orange.com/docs/5.3/about/license/

Reply:
Done in #2723